### PR TITLE
Add tutorial about how to create a starting kit

### PR DESCRIPTION
**Pull request by ZenanZha**
I just finish a Starting Kit, it was not a easy job...
I think adding these two file (one template, one tutorial) would greatly help others to understand what to do, when they are creating new Starting Kits.

### CODE REVIEW

**Comment by ZenanZha** [comment]
Should I pull the file as .md file instead of .html?

File: default/how_to_make_a_starting_kit/How(double click me).html [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+          <p class = "tip">
+            The template "tutorial.html" is given in the same folder as this file. :)
+          </p>
+        </li>
+
+        <!-- Step 3 -->
+        <li>
+          <h6>Create a Teaching Kit and Publish it.</h6>
+          <p>
+            This one is opectional.(because I saw a lot of people didn't do that...)
```
Comment (Rajat-dhyani) [inline default/how_to_make_a_starting_kit/How(double click me).html]: typo error : opectional --> optional

Comment (ZenanZha) [inline default/how_to_make_a_starting_kit/How(double click me).html]: Fixed, thank you.

**Comment by humphd** [comment]
This probably belongs in our DOCS vs. as an HTML file where you have it, and yes that likely means as a markdown file.  @flukeout could say more.

**Comment by flukeout** [comment]
Agreed with that @humphd - I think this would make a great addition to our Wiki pages. @ZenanZha could you take the content you've created and add it as a wiki page? Thanks!

**Pull request closed without merging**